### PR TITLE
Addition of Avalanche C-Chain Support

### PR DESCRIPTION
**Pull request by kylecbetts**
Avalanche is an open-source platform for launching decentralized applications and enterprise blockchain deployments in one interoperable, highly scalable ecosystem.

Avalanche C-Chain uses the CAIP-3 eip155 namespace with two chain IDs:
43113 for Avalanche Fuji Testnet
43114 for Avalanche Mainnet

### CODE REVIEW

**Comment by oed** [comment]
Hey @kylecbetts the `eip155` namespace is already supported by the "ethereum" AuthProvider. There should be no need to add any extra logic here!

**Comment by kylecbetts** [comment]
So should we still have an AvalancheAuthProvider class that is a subclass of EthereumAuthProvider with no logic, or are you suggesting adding nothing?

**Comment by oed** [comment]
@kylecbetts yeah, unless there is something that you do differently than the *EthereumAuthProvider* then there is no reason to add duplicate code :)

**Comment by kylecbetts** [comment]
@oed thanks for your patience, I have changed the implementation of AvalancheAuthProvider to simply inherit everything from EthereumAuthProvider.

**Comment by oed** [comment]
@kylecbetts I think you might have misunderstood. Seems like this PR is not needed at all if you can just use the *EthereumAuthProvider* directly? There is nothing added here in terms of functionality afaict.

**Comment by kylecbetts** [comment]
I guess I was just thinking for users of js-ceramic it would eliminate any confusion of wanting to use the Avalanche platform, as they might not know we use the same functionality as the EthereumAuthProvider. I agree nothing here in terms of functionality, just ease of use.

File: packages/blockchain-utils-linking/src/index.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 export * as ethereum from "./ethereum";
+export * as avalanche from "./avalanche";
 export * from "./util";
 export * as filecoin from "./filecoin";
 export * as polkadot from "./polkadot";
 export * as eosio from "./eosio";
 export * as cosmos from "./cosmos";
 
 export { EthereumAuthProvider } from "./ethereum";
+export { AvalancheAuthProvider } from "./avalanche";
```
Comment (oed) [inline packages/blockchain-utils-linking/src/index.ts:11]:
```suggestion
export { EthereumAuthProvider as AvalancheAuthProvider } from "./ethereum";
```

**Review by oed — CHANGES_REQUESTED** [review]
Ok if that's what we need, then I would just do the suggested change and remove everything else.

**Comment by kylecbetts** [comment]
@oed Sorry for all the confusion, I do appreciate the help.

**Review by oed — APPROVED** [review]
Thanks @kylecbetts

**Pull request merged**